### PR TITLE
Numerous changes and formating to fix gon

### PR DESCRIPTION
**Pull request by amigus**
This configuration works on my machine!

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,30 +27,25 @@ jobs:
...
+          sudo chmod +x /usr/local/bin/gon
 
       - name: Import GPG key
         id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v4
+        uses: crazy-max/ghaction-import-gpg@v5
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
-          fingerprint: ${{ secrets.GPG_FINGERPRINT }}
```
Comment (EndlessTrax) [inline .github/workflows/release.yml:42]: This needs to be added back in as a sub GPG key is being used.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -27,30 +27,25 @@ jobs:
...
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v4
         with:
           distribution: goreleaser
           version: latest
           args: release --clean
-          workdir: ./cmd/ssh-sign/
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
```
Comment (EndlessTrax) [inline .github/workflows/release.yml:53]: Same as above:

This needs to be added back in as a sub GPG key is being used.

File: .github/workflows/release.yml (hunk 1 above)
Comment (amigus) [reply .github/workflows/release.yml:42]: Done

File: .github/workflows/release.yml (hunk 2 above)
Comment (amigus) [reply .github/workflows/release.yml:53]: Likewise done.

**Review by EndlessTrax — APPROVED** [review]

**Pull request merged**